packages/api/src/repository/ResultRequest.ts

```diff
@@ -13,6 +13,7 @@ export class ResultRequestRepository {
 
   constructor (db: Db, _dataFeeds: Array<FeedInfo>) {
     this.collection = db.collection('result_request')
+    this.collection.createIndex({ feedFullName: 1 })
```

Comment:
In ResultRequest we are looking also for ` timestamp: { $gt: timestamp.toString() }` so the index should include it:

```suggestion
    this.collection.createIndex({ feedFullName: 1, timestamp: -1 })
```